### PR TITLE
DOC-12775: Update eventing compatibility copy.

### PR DESCRIPTION
**Pull request by ElliotFrancisHunter**
PR for ticket: https://jira.issues.couchbase.com/browse/DOC-12775

Resolving incorrect instructions due to implementation changes.

### CODE REVIEW

File: modules/ROOT/pages/server-compatibility-eventing.adoc [hunk 1]
```diff
@@ -53,12 +53,8 @@ You can use Eventing to handle data changes that happen when applications intera
 
 You can also use Eventing to generate vector embeddings from document fields, see xref:couchbase-lite:ROOT/cbl-whatsnew.adoc#vector-search[Vector Search] for more details about Vector Search with Couchbase Lite.
```
Comment (ElliotFrancisHunter) [inline modules/ROOT/pages/server-compatibility-eventing.adoc:53]: @simon-dew the commented section at Line 51. Is that functionality now available in 7.6.4 or has it been moved to 7.6.5?

**Review by RayOffiah — APPROVED** [review]
Yup, LGTM! 👍🏾

File: modules/ROOT/pages/server-compatibility-eventing.adoc [hunk 2]
```diff
@@ -67,9 +63,55 @@ For Sync Gateway versions that write import XATTRs:
...
 NOTE: If the `import_mutation_aware` flag is set to `true`, the performance of the Eventing function will drop because every mutation processed by Eventing will require a subdocument operation to maintain a cursor or progress for any Function that is sharing a Sync Gateway endpoint.  
 
+The procedure to enable Sync Gateway support for an Eventing function is as follows:
+
+. On Couchbase Server, xref:server:eventing:eventing-api.adoc#basic_pause[pause the function].
+
+. Using the Eventing REST API, set the `allow_sync_documents` setting for the function to false.
++
+NOTE: The `allow_sync_documents` setting is true by default.
+You must also include the `deployment_status` and `processing_status` settings in the request body.
```
Comment (simon-dew) [inline modules/ROOT/pages/server-compatibility-eventing.adoc]: I'm having some doubts as to whether `allow_sync_documents` _is_ true by default. The PRD says it is, but the spec in the Eventing repo says it isn't. Let's just delete this line for now. I'll correct the information in the REST API documentation when I know more.

```suggestion
You must also include the `deployment_status` and `processing_status` settings in the request body.
```

File: modules/ROOT/pages/server-compatibility-eventing.adoc (hunk 1 above)
Comment (simon-dew) [reply modules/ROOT/pages/server-compatibility-eventing.adoc:53]: No I think that functionality is available in 7.6.4 — you can uncomment those lines

File: modules/ROOT/pages/server-compatibility-eventing.adoc [hunk 3]
```diff
@@ -67,9 +63,55 @@ For Sync Gateway versions that write import XATTRs:
...
++
+. On Couchbase Server, xref:server:eventing:eventing-api.adoc#basic_resume[resume the function].
+
+For more details, see xref:server:eventing:eventing-api.adoc[Eventing REST API].
+
 == Using Eventing (Pre-Server 7.6.3)
 
-You can use Eventing and Sync Gateway connected to the same bucket, when Eventing operates on server buckets in *read only* mode -- see {server-eventing-terminologies--xref-bindings} for how to do this.
+You can use Eventing and Sync Gateway connected to the same bucket. 
+When Eventing operates on server buckets in *read only* mode -- see {server-eventing-terminologies--xref-bindings} for how to do this.
```
Comment (simon-dew) [inline modules/ROOT/pages/server-compatibility-eventing.adoc]: Not sure why you've split this into sentences? The second sentence doesn't seem to make sense by itself.

**Review by simon-dew — DISMISSED** [review]
Okay, I've got a couple of comments. Just a note, please do not merge this PR until Couchbase Server 7.6.4 is GA

File: modules/ROOT/pages/server-compatibility-eventing.adoc (hunk 1 above)
Comment (ElliotFrancisHunter) [reply modules/ROOT/pages/server-compatibility-eventing.adoc:53]: Done!

File: modules/ROOT/pages/server-compatibility-eventing.adoc (hunk 2 above)
Comment (ElliotFrancisHunter) [reply modules/ROOT/pages/server-compatibility-eventing.adoc]: Done!

File: modules/ROOT/pages/server-compatibility-eventing.adoc (hunk 3 above)
Comment (ElliotFrancisHunter) [reply modules/ROOT/pages/server-compatibility-eventing.adoc]: Ah it was a nit from Vale, reverted as I agree, doesn't make sense now I'm looking at it again

**Pull request merged**